### DISCUSSION TITLE
to_sql alias method?

### DISCUSSION THREAD

**Discussion by benkoshy** [jeremyevans/sequel#1953] [General]
Is it a ridiculous idea to have a `to_sql` method?

```rb
DB[:Orders].sql
# => SELECT * FROM `Orders`
DB[:Orders].to_sql # an alias
# => SELECT * FROM `Orders`
```

### Why?
My argument: `to_sql` is more ruby-esque, like `to_s`, `to_h`, or like ActiveRecord where `to_sql`  does the same thing, and is arguably more expessive.

I had tried the above `to_sql` and was surprised to see it not there. A google search turned up the right answer, but it prompted the above thought. my two cents.

**Comment by jeremyevans** [comment]
I'm generally against adding aliases, unless the current method name is wrong or misleading.  This is something that users can easily add by themselves if they want.

FWIW, I don't think `to_sql` is "more expressive".  It gives the impression you are converting the dataset to an SQL string, instead of obtaining the SQL that the dataset will use.  Considering that most datasets cache their SQL, you should think of the SQL as an attribute of the dataset, not a conversion from the dataset.
